source/examples/rapids-snowflake-cudf/notebook.ipynb

```diff
@@ -0,0 +1,356 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        "token": get_login_token(),
</pre><p>You could replace this line with <code>"token": pathlib.Path("/snowflake/session/token").read_text()</code> which would remove a little indirection while still being very readable.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>